Magitek/Logic/Warrior/Aoe.cs

```diff
@@ -121,17 +98,14 @@ public static async Task<bool> Orogeny()
 
         public static async Task<bool> PrimalRend()
         {
-            if (Core.Me.CurrentTarget == null)
+            if (!WarriorRoutine.ToggleAndSpellCheck(WarriorSettings.Instance.UseAoe, Spells.PrimalRend))
```

Comment:
I think Primal Rend is used even for single target, may be worth not disabling it even if the user has disabled UseAoe in settings? I suppose it depends why the user would disable aoe, if it's out of safety to not hit other mobs, then having it off here makes sense. If it's to do focused damage, then the user probably wouldn't want this. 

Tough choice!